### ISSUE TITLE
guide for MS SQL functions

### ISSUE THREAD

**Issue by Iffoffo** [dbeaver/dbeaver#18442]
**Is your feature request related to a problem? Please describe.**
Inside the SQL editor, is it possible to have the guide of a MS SQL function?

**Describe the solution you'd like**
if I write i.e. CONCAT, I'd like to have a popup with the guide for that function

**Describe alternatives you've considered**
AND/OR a link to a guide page (i.e. https://learn.microsoft.com/it-it/sql/t-sql/functions/concat-transact-sql )

**Comment by E1izabeth** [comment]
Could you describe in more details what do you mean by guide? Is it about function signature, or function description on a natural language, or examples of the function usages?

**Comment by Iffoffo** [comment]
I meant at least short description + syntax.

At the moment when you write i.e. "SELECT" it opens two popup with no infos.
![image](https://user-images.githubusercontent.com/21224215/205850352-9e3e6abb-00f0-4587-a6b3-eff7580692f4.png)

I was thinking something like "Eclipse PHP Development Tools".
![image](https://user-images.githubusercontent.com/21224215/205849954-2cc19db2-460f-4673-82b5-5fa94cb11083.png)

EXAMPLE: 
CONCAT 
This function returns a string resulting from the concatenation, or joining, of two or more string values in an end-to-end manner
CONCAT ( string_value1, string_value2 [, string_valueN ] )

**Comment by E1izabeth** [comment]
Thank you for explanation.

**Comment by Destrolaric** [comment]
@Iffoffo Hi, we already have this functionality. It is disabled by default and can be found in `Editors -> SQL Editor -> Code Completion -> Show server help topics`

**Comment by Iffoffo** [comment]
@Destrolaric Hi, if I enable that func. I only get the following:
![image](https://user-images.githubusercontent.com/21224215/205965715-3d3ec633-ea16-46fb-9398-8623d847da99.png)

Some other setup missing?

**Comment by Destrolaric** [comment]
@Iffoffo No, I, unfortunately, overlooked that it was MS SQL. That functionality, for now, is only for My SQL. But we probably may look to implement it in the future.
